### ISSUE TITLE
Team update - Mar 23, 2022

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/jupyter-a11y-mgmt#88]
This is a  @Quansight-Labs/czi-a11y-grant team sync 🎉🎉🎉! This is a way for the Team Members to provide status reports on what they've been up to this week and request help and attention for things they are working on. This issue will be closed at the end of the day.

_Copy and paste the template below, and answer questions as you wish!_

<details>
<summary>Response Template</summary>

```md
**Thanks I'd like to give 🙌**
- So-and-so helped me out a lot with XXX...
- Thanks for Jo's work on the XXX repo...

**Updates from last week :heavy_check_mark:**
- I worked towards deliverable: <link-to-deliverable>
- I had a meeting with ABC

**Challenges I faced and things I'd like assistance with 🙏**
- I had a hard time figuring out ...
- Could somebody take a look at ...

**My availability for next week**
- I'll be off on XXX day...
- I've got several meetings this week...
```

</details>

#### :mag: **Needs Triage**:

_The [Needs Triage issues](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/labels/status%3A%20needs%20triage) require an initial assessment and labeling._

* [Investigate accessibility statements](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/issues/52)

### :dart: Project boards

Please make sure the boards reflect the current status of the project.

[Jupyter a11y grant project](https://github.com/orgs/Quansight-Labs/projects/5/views/5)

**Comment by gabalafou** [comment]
**Updates from last week :heavy_check_mark:**
- No real updates yet, but coming soon!

**Challenges I faced and things I'd like assistance with 🙏**
- The usual... balancing commitments

**My availability for next week**
- I'll be available all next week

**Comment by gabalafou** [comment]
Meeting minutes:

- Gabriel gives updates. No updates since last week. Still balancing commitments.
- Isabela gives updates. Same as above, a few extra notes:
    - Gabriel asks for some more context about RIT. RIT is the Rochester Institute of Technology. It has a minor related to Open Source, which requires students to do some open source work. Isabela met with some students, created the issue above as a starting point for those students. Maybe something will come out of it, maybe not. But we have at least made a connection.
    - Isabela is working on fixing the [JupyterLab/Accessibility Docs](https://github.com/jupyter/accessibility/issues/75). She could use eyes on the issue linked above.
- Giving Gabriel the tea on the cancelation of the Jupyter accessibility workshop. tl;dr-- not enough people registered.
    - Longer discussion/brainstorming about how to create a collective audit, or some new kind of collective workshop (like the alt text workshop)
- Tony gives updates. Says he doesn't have any. Gabriel empathizes. Isabela asks how she can help.
- Long discussion with Gabriel about demands of Conda Store. 
    - tl;dr-- In order for Gabriel to meet demands of this a11y team, he needs to go from being actively involved with Conda Store to being passively involved (or not involved)
- Tony wonders if there should be more coordination between the people working on accessibility
